doc/classes/Image.xml

```diff
@@ -166,52 +175,56 @@
 			<argument index="0" name="color" type="Color">
 			</argument>
 			<description>
-				Fills an [code]Image[/code] with a specified [Color]
+				Fills this [code]Image[/code] with a specified [Color].
 			</description>
 		</method>
 		<method name="fix_alpha_edges">
 			<return type="void">
 			</return>
 			<description>
+				Blends the pixels with low alpha values with colors of nearby pixels.
 			</description>
 		</method>
 		<method name="flip_x">
 			<return type="void">
 			</return>
 			<description>
+				Flips the image horizontally.
 			</description>
 		</method>
 		<method name="flip_y">
 			<return type="void">
 			</return>
 			<description>
+				Flips the image vertically.
 			</description>
 		</method>
 		<method name="generate_mipmaps">
 			<return type="int" enum="Error">
 			</return>
 			<description>
+				Generates mipmaps for the image.
```

Comment:
- Might be helpful to describe what a mipmap is.
- Returns an Error - in what case?